src/device_models/common/variable_cost.jl

```diff
@@ -22,7 +22,7 @@ end
 
 function pwlgencost(m::JuMP.Model, variable::VariableRef, cost_component::Array{Tuple{Float64, Float64}})
 
-    pwlvars = @variable(m, [i = 1:(length(cost_component)-1)], base_name = "pwl_{$(variable)}")
+    pwlvars = @variable(m, [i = 1:(length(cost_component)-1)], base_name = "pwl_{$(variable)}", start = 0.0, lower_bound = 0.0, upper_bound = (cost_component[i+1][1] - cost_component[i][1]))
```

Comment:
See #79 

cc @claytonpbarrows

Reply:
I think this can't have the bounds. I'm not exactly sure why, but it creates infeasible problems.